tests/core/BearTest.py

```diff
@@ -254,3 +258,18 @@ def test_json(self):
             'REQUIREMENTS': set()}
 
         self.assertEqual(result, expected)
+
+    def test_class_bear_deps_immutability(self):
+        section = Section('test-section')
+        uut = BearWithDependencies(section, {})
+
+        self.assertNotEqual(id(uut.BEAR_DEPS),
```

Comment:
Are you trying to check if belong to possibly the same memory address with id?

Reply:
yes

Reply:
people shall be able to modify `self.BEAR_DEPS` without modifying `cls.BEAR_DEPS`, so a copy is made. To distinct the lists, I use `id` here.

Reply:
Oh my bad, why didn't I thought of using `assertIs`/`assertIsNot`...